workflow/landing-page/js/loader.js

```diff
@@ -0,0 +1,71 @@
+//Кнопки пагинации
```

Comment:
https://www.youtube.com/watch?v=bUa7xJWuz_s

Reply:
I've played this game)